### PR TITLE
Allow all origins for /refresh endpoints

### PR DESCRIPTION
**Pull request by tatlax3636**
enable all origins to access /refresh endpoints for collectors

### CODE REVIEW

**Review by rvema — CHANGES_REQUESTED** [review]
please increment the version in pom.xml to `3.15.0`

**Review by rvema — CHANGES_REQUESTED** [review]
please address this minor change

File: src/main/java/com/capitalone/dashboard/config/WebSecurityConfig.java [hunk 1]
```diff
@@ -35,6 +35,7 @@ protected void configure(HttpSecurity http) throws Exception {
         http.csrf().disable()
                 .authorizeRequests()
                 .antMatchers("/ping").permitAll()
+                .antMatchers("/refresh").permitAll()
```
[the remark this answers is not in the document]
Comment (tatlax3636) [reply src/main/java/com/capitalone/dashboard/config/WebSecurityConfig.java]: Updated

**Review by rvema — APPROVED** [review]
lgtm

**Pull request merged**